README.md

```diff
@@ -30,33 +31,112 @@ global:
+* [`handlebars`](https://github.com/altano/handlebars-loader): Loads handlebars template and returns a function
+* [`ractive`](https://github.com/rstacruz/ractive-loader): Pre-compiles Ractive templates for interactive DOM manipulation
+* [`jsx`](https://github.com/petehunt/jsx-loader): Transform jsx code for [React](http://facebook.github.io/react/) to js code.
+* [`em`](https://github.com/yoshdog/emblem-loader): Compiles [Emblem](http://emblemjs.com/) to Handlebars.js
+* [`ejs`](https://github.com/okonet/ejs-loader): Loads EJS ([underscore](http://underscorejs.org/#template)( templating engine) template and returns a pre-compiled function
+* [`markdown`](https://github.com/peerigon/markdown-loader): Compiles Markdown to HTML
+* [`ng-cache`](https://github.com/teux/ng-cache-loader): Puts HTML partials in the Angular's $templateCache
+
+**styling**
+* [`bootstrap-sass`](https://github.com/justin808/bootstrap-sass-loader): Loads a configuration file for Twitter Bootstrap integration using Sass. Allows complete customization via Sass.
+* [`style`](https://github.com/webpack/style-loader): Add exports of a module as style to DOM
+* [`css`](https://github.com/webpack/css-loader): Loads css file with resolved imports and returns css code
+* [`less`](https://github.com/webpack/less-loader): Loads and compiles a less file
+* [`sass`](https://github.com/jtangelder/sass-loader): Loads and compiles a scss file
+* [`stylus`](https://github.com/shama/stylus-loader): Loads and compiles a stylus file
+* [`rework`](https://github.com/okonet/rework-loader): Post-process CSS with [Rework](https://github.com/reworkcss/rework) and returns CSS code
+* [`postcss`](https://github.com/postcss/postcss-loader): Post-process CSS with Autoprefixer and [other PostCSS plugins](https://github.com/postcss/postcss#built-with-postcss)
+* [`autoprefixer`](https://github.com/passy/autoprefixer-loader): Add vendor prefixes to CSS rules using values from Can I Use
+* [`namespace-css`](https://github.com/jeffling/namespace-css-loader): Namespace your css with a given selector (for encapsulating all rules in one subset of your site)
+
+**translation**
+* [`po`](https://github.com/dschissler/po-loader): Loads a PO gettext file and returns JSON
+
+**support**
+* [`mocha`](https://github.com/webpack/mocha-loader): do tests with mocha in browser or node.js
+* [`eslint`](https://github.com/MoOx/eslint-loader): PreLoader for linting code using ESLint.
+* [`jshint`](https://github.com/webpack/jshint-loader): PreLoader for linting code.
+* [`jscs`](https://github.com/unindented/jscs-loader): PreLoader for style checking.
+* [`injectable`](https://github.com/jauco/webpack-injectable): Allow to inject dependencies into modules
+* [`transform`](https://github.com/webpack/transform-loader): Use browserify transforms as loader.
+* [`image-size`](https://github.com/patcoll/image-size-loader): Loads an image and returns its dimensions and type
```

Comment:
I think this section should be called `code quality`. Also, I am not clear on the purpose of the `image-size` loader.